docs/deploy-seperate-pods.md

```diff
@@ -17,12 +17,17 @@ This configuration has several advantages:
     - `contourcerts`: must contain `contourcert.pem` and `contourkey.pem` keys that contain a certificate and key for Contour. The certificate must be valid for the name `contour` either via CN or SAN.
     - `envoycerts`: must contain `envoycert.pem` and `envoykey.pem` keys that contain a certificate and key for Envoy.
 
+For detailed instructions on how to configure the required certs manually, see the [step-by-step TLS HOWTO][2].
+
 ## Deploy Contour
 
 1. [Clone the Contour repository][1] and cd into the repo.
-2. Ensure you have `openssl` installed (or at least something that provides that command.`)
-2. Generate the keypairs required to secure the TLS connection between Contour and Envoy. You can do that using `contour gencerts --pem` and then `make applycerts`, assuming that your kubectl will connect to the cluster you want. If not, follow the [detailed generation directions][2]. 
-3. Run `kubectl apply -f examples/ds-hostnet-split/`
+2. Run `kubectl apply -f examples/ds-hostnet-split/`
```

Comment:
What's the upgrade path for someone who had previously deployed this example in their 0.13 cluster? Can they kubectl apply -f again; pick up the new contour image version and the certificate job will also deploy?

If this is a bit sketchy maybe we should clone ds-hostnet-split, add a new example, then we can deprecate ds-hostnet-split in 0.15?

Reply:
I just confirmed that the upgrade path is just `kubectl -f apply` again - although I just noticed I forgot to change the certgen image spec to `gcr.io/heptio-images/contour:master`, I'll fix that.

If we want to change the Job spec later, we may need to do something clever, as Job Pod specs are immutable. But this works just fine for upgrading from no security to with security.